packages/common/container-definitions/src/deltas.ts

```diff
@@ -212,6 +212,11 @@ export interface IDeltaManager<T, U>
 	 */
 	readonly serviceConfiguration: IClientConfiguration | undefined;
 
+	/**
```

Comment:
this is a pretty large change that cuts across multiple compatibility layers. is there a design for it? i also think it should be done in an isolated change. + @agarwal-navin who has been thinking about this space.

Reply:
I would also like to see stronger typing used here to avoid mistakes.